kubejs/server_scripts/normal/unification/unify_ore_processing.js

```diff
@@ -196,7 +196,9 @@ function metal_ore_processing(material, properties, event, id_prefix) {
     // Fallback output checks:
     if (crushed_ore_itemStack.isEmpty()) {
         crushed_ore_itemStack = Item.of(Ingredient.of(`#create:crushed_ores/${material}`).getItemIds()[0]);
-        crushed_ore_itemStack = (crushed_ore_itemStack.isEmpty() || crushed_ore_itemStack.getId() == "minecraft:barrier")? null: crushed_ore_itemStack;
+        if (crushed_ore_itemStack.isEmpty() || crushed_ore_itemStack.getId() == "minecraft:barrier") {
```

Comment:
There were several similar issues in this file below this I'm afraid 😄

Reply:
Woops! Time to check the entire file for those then 😅